hw/ip/hmac/README.md

```diff
@@ -15,41 +15,42 @@ See that document for integration overview within the broader OpenTitan top leve
 
 ## Features
 
-- HMAC supporting multiple digest sizes: SHA-2 256/384/512 hashing algorithm
-- HMAC-SHA-2 and unkeyed SHA-2 dual mode
+- Dual modes: HMAC based on SHA-2 and unkeyed SHA-2
```

Comment:
nit: I would drop the `unkeyed` term here. I think SHA-2 clearly refers to the hash function and in the standard definition there is no key input for a hash function.

Reply:
Maybe:
```suggestion
- Dual modes: SHA-2 / HMAC based on SHA-2
```
?